system/jlib/jlzw.cpp

```diff
@@ -2622,7 +2632,12 @@ class CAESCompressor : implements ICompressor, public CInterface
         return true;
     }
 
-    void close()
+    virtual bool supportsBlockCompression() const override { return false; }
+    virtual bool supportsIncrementalCompression() const override { return true; }
+
+    virtual size32_t compressBlock(size32_t destSize, void * dest, size32_t srcSize, const void * src) override { return 0; }
```

Comment:
Should this be throwUnexpected() given that supportsBlockCompression() returns false?

Reply:
0 is fine - it means it failed to compress